lms/migrations/versions/e13fb37c96e5_backfill_lms_user_application_instance.py

```diff
@@ -0,0 +1,43 @@
+"""Backfill lms_user_application_instance."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "e13fb37c96e5"
+down_revision = "f61cb94edfc8"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    conn.execute(
+        sa.text(
+            """
+        WITH backfill as (
+            SELECT
+                "user".created,
+                "user".updated,
+                lms_user.id lms_user_id,
+                "user".application_instance_id
+            FROM "user"
+            JOIN lms_user on lms_user.h_userid = "user".h_userid
+        )
+        INSERT INTO lms_user_application_instance (
```

Comment:
This was probably possible the same migration where we filled LMSCourse.

Doing it a separate one for more clarity